README.md

```diff
@@ -63,8 +63,16 @@ Jump to [FAQ](#faq).
 ## Installing LLVM
 `package:ffigen` uses LLVM. Install LLVM (9+) in the following way.
 
-#### ubuntu/linux
-1. Install libclangdev - `sudo apt-get install libclang-dev`.
+#### Linux
+1. Install libclangdev.
+
+##### Ubuntu
```

Comment:
Maybe omit the Ubuntu/Fedora headers and just make it:

```
#### Linux

1. Install libclangdev.
   With apt-get: `sudo apt-get install libclang-dev`.
   With dnf: `sudo dnf install clang-devel`.
```

#### Linux

1. Install libclangdev.
   With apt-get: `sudo apt-get install libclang-dev`.
   With dnf: `sudo dnf install clang-devel`.